x-pack/plugins/observability/docs/openapi/slo/components/schemas/filter_meta.yaml

```diff
@@ -0,0 +1,27 @@
+title: FilterMeta
+description: Defines properties for a filter
+type: object
+properties:
+  alias:
+    type: string
+    nullable: true
+  disabled:
+    type: boolean
+  negate:
+    type: boolean
+  controlledBy:
+    type: string
+  group:
+    type: string
+  index:
+    type: string
+  isMultiIndex:
+    type: boolean
+  type:
+    type: string
+  key:
+    type: string
+  params:
+    type: object
+  value:
+    type: string
```

Comment:
Can we add examples/enum values for every params because If I were to integrate with this API, there is absolutely zero chance I make it work 😅

Reply:
i think those or just boolean/string values, i don't think examples would make sense. 

if user wants to use API, best way would be ask them to copy examples via UI.